leaf/extensions/image.py

```diff
@@ -0,0 +1,220 @@
+import enum
+
+import PIL.Image
+import aiohttp
+from discord.ext import commands
+from discord import app_commands
+import discord
+from bot import LeafBot
```

Comment:
Remember to follow PEP 8 order, standard library imports first, then local imports, then third-party libraries.